Dockerfile

```diff
@@ -10,6 +10,8 @@ COPY src/update_xs_yum.ml tmp/src/
 
 WORKDIR tmp/
 
+RUN opam update
```

Comment:
I still get the error that I assume you saw too, even with `RUN opam update`:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - ppx_tools.5.0
        (This may be fixed by running 'opam update')
```

Reply:
Yep, I saw that, probably an issue with the opam repo :S